### ISSUE TITLE
[Crash Report]: App crashes when going back from main player screen after a different main player style was selected

### ISSUE THREAD

**Issue by ok-c0ol** [chr56/Phonograph_Plus#99]
### Crash log

```
Crash Report:

Package name:    player.phonograph.plus
App version:     0.7.2.1 (621)
Git Commit Hash: 33ce6fca472087b9d01b
Android version: 13 (SDK 33)
Device brand:    google  (by Google)
Device model:    Pixel 6 (code: oriole)
Product name:    oriole
Build version:   TQ3A.230605.010 
                 (10121037)
Hardware:        oriole
Language:        en

java.lang.NullPointerException
	at player.phonograph.ui.fragments.player.card.CardPlayerFragment.onBackPressed(SourceFile:3)
	at player.phonograph.ui.activities.base.AbsSlidingMusicPanelActivity.handleBackPress(SourceFile:17)
	at player.phonograph.ui.activities.MainActivity.handleBackPress(Unknown Source:41)
	at player.phonograph.ui.activities.base.AbsSlidingMusicPanelActivity.onBackPressed(Unknown Source:0)
	at android.app.Activity.onKeyUp(Activity.java:3921)
	at android.view.KeyEvent.dispatch(KeyEvent.java:2925)
	at android.app.Activity.dispatchKeyEvent(Activity.java:4264)
	at androidx.core.app.r.superDispatchKeyEvent(Unknown Source:0)
	at androidx.core.view.q.f(SourceFile:11)
	at androidx.core.app.r.dispatchKeyEvent(Unknown Source:18)
	at androidx.appcompat.app.n.dispatchKeyEvent(Unknown Source:22)
	at androidx.appcompat.view.r.dispatchKeyEvent(Unknown Source:2)
	at androidx.appcompat.app.c0.dispatchKeyEvent(Unknown Source:21)
	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:402)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:6708)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:6574)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6034)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6091)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6057)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6222)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6065)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6279)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6038)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6091)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6057)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6065)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6038)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6091)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6057)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6255)
	at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:6435)
	at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3649)
	at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:3169)
	at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:3160)
	at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3626)
	at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:154)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:335)
	at android.os.Looper.loopOnce(Looper.java:161)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7918)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)


```

### Steps to reproduce the bug

1. Enter Settings 
2. Select a different Main Player Style (from my testing, it doesn't matter which one, but it must be different than the currently configured one)
3. Back out of settings
4. Tap on main screen to remove sidebar 
5. Open the main player
6. Back out of the main player. The crash happens here.

### Other helpful information

Thanks for such a great FOSS music app -  I just started using it yesterday and I love it

### Version

0.7.2.1

**Comment by chr56** [comment]
It should be fixed in 084d468a5197f5513855550dbb4406d6d12c1a9c.

**Comment by chr56** [comment]
Fixed in both [v0.7.2.2(lts)](https://github.com/chr56/Phonograph_Plus/releases/tag/v0.7.2.2) and [v1.0.0](https://github.com/chr56/Phonograph_Plus/releases/tag/v1.0.0)! Close!
